keyboards/jacky_studio/piggy60/info.json

```diff
@@ -45,6 +46,7 @@
         "tap_capslock_delay": 10,
         "term": 170
     },
+    "community_layouts": ["60_hhkb", "60_tsangan_hhkb"],
```

Comment:
```suggestion
```
The `solder` info.json will override this; place it in the hotswap one instead.

Reply:
> the keyboard-level definition takes precedence over the revision-level definition. Not sure if intended behavior.

It's not intended, but fixing it may break other stuff that assumed this behaviour. We (I) need to set aside some time for it.

For now, if you want to "extend" the layouts of a keyboard, they must all be placed in each revision folder instead of having the common ones at the root.